### DISCUSSION TITLE
support MQTT discovery for HomeAssistant

### DISCUSSION THREAD

**Discussion by faceprint** [teslamate-org/teslamate#493] [Ideas]
Setting up the MQTT integration with HomeAssistant can be tedious.  HomeAssistant supports a MQTT discovery protocol, described here:

https://www.home-assistant.io/docs/mqtt/discovery/

teslamate should publish homeassistant discovery info, so sensors could be automatically created in HomeAssistant

**Comment by nebhale** [comment]
Now implemented: https://github.com/teslamate-org/teslamate/releases#release-v4.1.0
